packages/flutter/lib/src/material/refresh_indicator.dart

```diff
@@ -261,6 +261,9 @@ class RefreshIndicatorState extends State<RefreshIndicator> with TickerProviderS
     super.dispose();
   }
 
+  // [dragDetails] is non-null means that the scroll notifications are emitted because of a drag.
+  // The indicator can only be triggered by dragging, and cannot be triggered
+  // by other non-dragging behaviors such as [ScrollController.jumpTo].
```

Comment:
Can you add more detail on what does it mean when
```
(notification is ScrollStartNotification && notification.dragDetails != null)
```
is true

and 

```
notification is ScrollUpdateNotification && notification.dragDetails != null
```
is true.